installer/install_dependencies.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Enable docker repository
+echo Enabling docker repository
+mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+# Update local repositories
+echo Updating local repositories
+apt-get update
+
+# Install dependencies
+echo Installing dependencies
+apt-get install -y apt-transport-https ca-certificates gnupg gnupg-agent lsb-release software-properties-common sshpass pv gawk
+
+# Install python dependencies
+echo Installing Python dependencies
+apt-get install -y python3-pip
+
+# Install ansible if not present
+if [ "`which ansible`" != "" ]; then
+    echo ansible already installed, skipping.
+else
+    echo Installing ansible
+    python3 -m pip install ansible
+fi
+
+# Install docker if not present
+if [ "`which docker`" != "" ]; then
+    echo Docker already installed, skipping.
+else
+    echo Installing docker
+    apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
+fi
+
+# Install Go if not present
+if [ "`which go`" != "" ]; then
+    IFS=' '
+    v=`go version | { read _ _ v _; echo ${v#go}; }`
+    IFS='.'
+    read -ra v <<< "$v"
+    if (( ${v[0]} == 1 && ${v[1]} >= 17 )); then
+        echo Go 1.17+ already installed, skipping.
+    else
+        echo Removing existing Go installation
```

Comment:
@devanshjain7 @PravinRanjan10 User may have existing Go installation for some other work.
Before deleting the existing Go installation confirm with the user if it is ok. 
Or we need to install the required Go version separately and use it.